ietf/doc/views_doc.py

```diff
@@ -303,34 +290,23 @@ def document_main(request, name, rev=None, document_html=False):
         exp_comment = doc.latest_event(IanaExpertDocEvent,type="comment")
         iana_experts_comment = exp_comment and exp_comment.desc
 
-        # Do not show the Auth48 URL in the "Additional URLs" section
-        additional_urls = doc.documenturl_set.exclude(tag_id='auth48')
-
         html = None
         js = None
         css = None
         diff_revisions = None
         simple_diff_revisions = None
         if document_html:
-            diff_revisions=get_diff_revisions(request, name, doc if isinstance(doc,Document) else doc.doc)
+            diff_revisions=get_diff_revisions(request, name, doc)
```

Comment:
Instead of would mean hitting the database before rejection. (the test would have to be after the get_object_or_404 before we could inspect type_id.)

I don't think "in addition to" would really buy us anything other than an assert like check that we haven't let a document of type RFC slip in that has a name that starts with something other than rfc.

So, how about a logassertion instead?